packages/core/src/logger.ts

```diff
@@ -62,7 +63,10 @@ export class Logger<
       return envelopedCallback();
     }
 
-    const newContext: Partial<TTaskContext> = context ? snapshot.v8(context) : {};
+    // @ts-expect-error Dont know...
+    const newContext: Partial<TTaskContext> = context ? snapshot.v8(context) : {
+      taskStack: [v4()],
+    };
```

Comment:
aha, už to vidím - ale v tom případě bys tady měl do `taskStack` přidat nový UUID, pokud startuješ subtask, nikde jinde to nevidím, correct?

Reply:
pridano

Reply:
hmm, jako fungovat to bude, ale je to trochu fugly.. co spíš třeba takhle?

```typescript
const newContext: Partial<TTaskContext> = context ? snapshot.v8(context) : { taskStack: [] };
newContext.taskStack.push(v4());
```

Reply:
deal